app/credentials.js

```diff
@@ -1,48 +1,32 @@
-/* global require, module */
+/* global api */
 
-const settings = require('electron-settings');
-const keytar = require('keytar')
-
-function addCredentials (connectionManager, login, password) {
+async function addCredentials(connectionManager, login, password) {
     const xmppService = login.split('@').pop()
-    settings.setSync('connectionManager', connectionManager)
-    settings.setSync('login', login)
-    keytar.setPassword(xmppService, login, password)
+    await api.settings.set('connectionManager', connectionManager)
```

Comment:
These calls cause the following error for me:

`Uncaught (in promise) Error: Error invoking remote method 'settings': No handler registered for 'settings'`

Reply:
Looks like moving the `icpMain.handle` lines higher up in main.js solves the issue...